post-scan-actions/azure-python-slack-notification/template.json

```diff
@@ -0,0 +1,233 @@
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "fssSlackFunctionName": {
+            "type": "string",
+            "defaultValue": "[concat('fssslack', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "The name of the slack function to make."
+            }
+        },
+        "webHookURL": {
+            "type": "string",
+            "defaultValue": "[concat('fssslack', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "Slack Webhook URL"
+            }
+        },
+        "slackChannelName": {
+            "type": "string",
+            "defaultValue": "[concat('fssslack', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "Name of Slack Channel"
+            }
+        },
+        "subscriptionName": {
+            "type": "string",
+            "defaultValue": "[concat('fssslack', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "Name of Azure Subscription"
+            }
+        },
+        "scanResultTopicResourceID": {
+            "type": "String",
+            "metadata": {
+                "description": "The resource ID of the scan result topic Server Bus topic in storage stack."
+            }
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "functionAppName": "[parameters('fssSlackFunctionName')]",
+        "hostingPlanName": "[parameters('fssSlackFunctionName')]",
+        "applicationInsightsName": "[parameters('fssSlackFunctionName')]",
+        "storageAccountName": "[concat('fssslack', uniquestring(resourceGroup().id))]",
+        "scanResultTopicResourceIDSplits": "[split(parameters('scanResultTopicResourceID'), '/')]",
+        "scanResultTopicResourceSubscriptionID": "[variables('scanResultTopicResourceIDSplits')[2]]",
+        "scanResultTopicResourceGroupName": "[variables('scanResultTopicResourceIDSplits')[4]]",
+        "scanResultTopicNamespace": "[variables('scanResultTopicResourceIDSplits')[8]]",
+        "scanResultTopicName": "[variables('scanResultTopicResourceIDSplits')[10]]",
+        "scanResultTopicNamespaceResourceID": "[split(parameters('scanResultTopicResourceID'), '/topics/')[0]]",
+        "scanResultSASKeyName": "RootManageSharedAccessKey",
+        "scanResultAuthRuleResourceID": "[format('{0}/authorizationRules/{1}', variables('scanResultTopicNamespaceResourceID'), variables('scanResultSASKeyName'))]",
+        "scanResultTopicSubscriptionName": "fss-slack-subscription",
+        "scanResultTopicAPIVersion": "2017-04-01",
+        "functionAppAPIVersion": "2020-12-01",
+        "fssSlackRoleAssignmentDeployment": "fssSlackRoleAssignment",
+        "slackWebHook": "[parameters('webhookURL')]",
+        "slackChannel": "[parameters('slackChannelName')]",
```

Comment:
The variables values are exactly the same as the parameters values, so you can directly use them below by `[parameters('xxx')]` without creating variables.
```suggestion
```